attribute-service-impl/src/test/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModelTest.java

```diff
@@ -353,4 +355,151 @@ void testScopeStringCompatibility() throws IOException {
     expected = template.toBuilder().setScopeString("NEWSCOPE").setId("NEWSCOPE.key").build();
     Assertions.assertEquals(expected, AttributeMetadataModel.fromJson(inputJson).toDTO());
   }
+
+  @Test
+  void testCustomWithoutTenantId() throws IOException {
+    final String json =
+        "{"
+            + "\"fqn\":\"fqn\","
+            + "\"key\":\"key\","
+            + "\"materialized\":true,"
+            + "\"unit\":\"ms\","
+            + "\"type\":\"ATTRIBUTE\","
+            + "\"labels\":[\"item1\"],"
+            + "\"groupable\":true,"
+            + "\"supportedAggregations\":[],"
+            + "\"onlyAggregationsAllowed\":false,"
+            + "\"sources\":[],"
+            + "\"definition\":{\"projection\":{\"attributeId\":\"test\"}},"
+            + "\"internal\":true,"
+            + "\"id\":\"EVENT.key\","
+            + "\"value_kind\":\"TYPE_STRING\","
+            + "\"display_name\":\"Some Name\","
+            + "\"scope_string\":\"EVENT\""
+            + "}";
+
+    final AttributeMetadata expectedMetadata =
+        AttributeMetadata.newBuilder()
+            .addLabels("item1")
+            .setFqn("fqn")
+            .setKey("key")
+            .setId("EVENT.key")
+            .setDisplayName("Some Name")
+            .setMaterialized(true)
+            .setGroupable(true)
+            .setScope(AttributeScope.EVENT)
+            .setScopeString(AttributeScope.EVENT.name())
+            .setType(AttributeType.ATTRIBUTE)
+            .setUnit("ms")
+            .setValueKind(AttributeKind.TYPE_STRING)
+            .setDefinition(
+                AttributeDefinition.newBuilder()
+                    .setProjection(Projection.newBuilder().setAttributeId("test"))
+                    .build())
+            .setInternal(true)
```

Comment:
I don't think this test is a valid case, but if we did want to keep it - setCustom(false) would be appropriate here. It's equivalent to the default boolean value which is why the test is passing, but omitting it in one test and not the other indicates to the reader we expect different behavior.